code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -234,7 +238,14 @@
 				if(org.status & LIMB_DESTROYED)
 					status = "MISSING!"
 
-				to_chat(src, "\t [status=="OK"?"<span class='notice'> ":"<span class='warning'> "]My [org.display_name] is [status].</span>")
+				if(brute_treated == 0 & brutedamage > 0)
```

Comment:
`brute_treated` is a boolean value (it can only be true or false, zero being false, one being true), so instead of using magic numbers do either `brute_treated == FALSE` or just simply `!brute_treated`.

Same for the other instances here of `bool == 0`.